src/main/config/azure-blob-application.properties

```diff
@@ -30,10 +30,14 @@ yarn.package.path=file://${basedir}/target/${project.artifactId}-${pom.version}-
 # StreamApplication class
 app.class=samza.examples.azure.AzureBlobApplication
 
-#Azure blob essential configs
+# Azure blob essential configs
 systems.azure-blob-container.samza.factory=org.apache.samza.system.azureblob.AzureBlobSystemFactory
 sensitive.systems.azure-blob-container.azureblob.account.name=your-azure-storage-account-name
 sensitive.systems.azure-blob-container.azureblob.account.key=your-azure-storage-account-key
 
+# Config Loader
+job.config.loader.factory=org.apache.samza.config.loaders.PropertiesConfigLoaderFactory
+job.config.loader.properties.path=./__package/config/azure-blob-application.properties
```

Comment:
Do you think it might be confusing to users what `__package` refers to? That is kind of a YARN implementation detail. Maybe documentation can help clarify, but if you can find a programatic way to hide that implementation detail, that would be nice too.

Reply:
Yes, "__package" is Samza's internal implementation detail where localized job to be unzipped at.

It is challenging to do it programmatically as clients may or may not include __package in the path, and different programs may have different locations to put their configs.